keepercommander/importer/importer.py

```diff
@@ -138,17 +141,56 @@ def get_folder_path(self):
         return path
 
 
+class RecordField:
+    def __init__(self):
+        self.type = None    # type: Optional[str]
+        self.label = None   # type: Optional[str]
+        self.value = None   # type: any
+
+    def name_key(self):
+        if self.type and self.label:
+            return f'${self.type.lower()}.{self.label.lower()}'
+        elif self.type:
+            return f'${self.type.lower()}'
+        else:
+            return (self.label or '').lower()
+
+    def hash_key(self):
+        value = self.value
+        if value:
+            if type(value) is str:
+                pass
+            elif type(value) is list:
+                value = '|'.join((str(x) for x in value))
```

Comment:
A tiny thing, but I don't think it's necessary to double the parens here.  The following should work fine:
value = '|'.join(str(x) for x in value)

Reply:
Thanks. It is good to know.